src/toil/jobStores/aws/jobStore.py

```diff
@@ -47,6 +49,43 @@
 
 log = logging.getLogger(__name__)
 
+def copy_key_multipart(srcBucket, srcKey, dstBucket, dstKeyName, part_size=None, headers={}):
```

Comment:
I was told to make that variable in that function snake case. Is that still the case @hannes-ucsc ?